mne_bids/write.py

```diff
@@ -282,8 +282,13 @@ def _participants_tsv(raw, subject_id, fname, overwrite=False,
         _old_keys = data.keys()
         data = _combine(orig_data, data, 'participant_id')
 
-        # get the keys that were new
-        for key in set(orig_data.keys()) - set(_old_keys):
+        # loop through the keys in the original data to
+        # keep the original order, and add back
+        # the data in participants files that user added
```

Comment:
```suggestion
        # Loop through all keys in the original data to keep the original
        # order, and insert the data that user has added to their
        # participants files.
```

Maybe easier to read that way? :)